<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>824</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1277</width>
    <height>824</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1277</width>
    <height>854</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YES! Music</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ResourseMusic.qrc">
    <normaloff>:/buttons/logo.png</normaloff>:/buttons/logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget_lists">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>150</y>
      <width>401</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_playlist">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QListWidget" name="listWidget_playlist">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>401</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
color:rgb(50,50,50);
</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_fav_list"/>
    <widget class="QWidget" name="page_history">
     <widget class="QListWidget" name="listWidget_history">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>401</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
color:rgb(50,50,50);
</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_playlist">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>90</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/buttons/playlist.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_history">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>90</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/buttons/history.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_file">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/buttons/add_file.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_previous">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border-image:url(:/buttons/previous.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>730</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/buttons/play.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>730</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/buttons/next.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_progress">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>750</y>
      <width>621</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mode">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>730</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/buttons/loop.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_volume">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>730</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/buttons/volumn.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_info">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>700</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_time">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>700</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_file">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>600</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_volume">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>660</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_all">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_refresh">
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>611</width>
        <height>451</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent</string>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::WidgetWidth</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>10</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/buttons/close.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/buttons/logo.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pause">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>730</y>
      <width>45</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/buttons/pause.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ResourseMusic.qrc"/>
 </resources>
 <connections/>
</ui>
